Changelog.md

```diff
@@ -1,6 +1,7 @@
 ### 0.4.10 (unreleased)
 
 Features:
+ * Add ``assert(condition)`` to abort execution.
```

Comment:
I can change these late today. Can you approve it? Would like to merge it before the weekend.